build.gradle

```diff
@@ -421,7 +421,7 @@ subprojects {
 
   tasks.withType(Test) {
     // If GRADLE_MAX_TEST_FORKS is not set, use half the available processors
-    maxParallelForks = (System.getenv('GRADLE_MAX_TEST_FORKS') ?: (Runtime.runtime.availableProcessors().intdiv(2) ?: 1)).toInteger()
+    maxParallelForks = 1
```

Comment:
did you mean to change this?

Reply:
Just did that to see whether that makes the acc tests timing out pass ...
Fixed!